src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -9,6 +9,8 @@
     <DebugSymbols>true</DebugSymbols>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
 
+    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">$(MicrosoftNETCoreAppPackageVersion)</RuntimeFrameworkVersion>
```

Comment:
I think this still causes problems, FWIW. Whatever is latest runtime version is likely to be ahead of what is in the shared runtime in core-sdk, so this will break.